### PR TITLE
NanoUIfication

### LINKED ISSUE #1484: Chem Master and pill bottles
#### Brief description of the issue:
The empty pill bottles the chemists get will not accept pills from the chem master, they can be manually placed inside however.
On the chem master's interface, the capacity of the pill bottle will read 'current number of pills/'

#### What you expected to happen:
For it to accept pills from the chem master.

#### What actually happened:
No pills could be placed inside the pill bottle from the chem master.

#### Steps to reproduce:
1. Get a empty pill bottle.
2. Get reagents.
3. Place both inside the chem master.
4. Try to put the reagent pills into the pill bottle from the chem master interface.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.

Server revision: master - 2016-04-22
134157feb579191f6429c479cb61572b67261971 
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

### PR DESCRIPTION
**Pull request by SinTwo**
For pictures of the UI's, see album below.
http://imgur.com/a/gGqpe

Advanced Body Scanner NanoUI was shameless pulled from https://github.com/ParadiseSS13/Paradise/pull/2393 and then cleaned up to be compatible with our things and stuff.

EDIT: Fixes #1484 and #1964

### CODE REVIEW

**Comment by SinTwo** [comment]
Unit test failure is actually due to unit test failure and not code failure. It is detecting two <font> tags that are inside a conditional, in actuality only one <font> tag is ever actually open and as such is closed by the </font> tag later; however, the unit test doesn't take the conditional into account and thinks that two <font> tags are opened and only one is ever closed.

File: code/game/machinery/adv_med.dm [hunk 1]
```diff
@@ -8,141 +8,141 @@
 	icon = 'icons/obj/Cryogenic2.dmi'
 	icon_state = "body_scanner_0"
 	density = 1
+	dir = 8
```
Comment (PJB3005) [inline code/game/machinery/adv_med.dm]: Use `WEST`

File: code/game/machinery/adv_med.dm [hunk 2]
```diff
@@ -8,141 +8,141 @@
...
+	light_color = "#00FF00"
+
 /obj/machinery/bodyscanner/New()
 	..()
+	spawn( 5 )
+		var/obj/machinery/body_scanconsole/C = locate(/obj/machinery/body_scanconsole) in range(2,src)
+		if(C)
+			C.connected = src
+		return
+	return
```
Comment (PJB3005) [inline code/game/machinery/adv_med.dm]: Trailing return.

File: code/game/machinery/adv_med.dm [hunk 3]
```diff
@@ -8,141 +8,141 @@
...
 
+	light_color = "#00FF00"
+
 /obj/machinery/bodyscanner/New()
 	..()
+	spawn( 5 )
+		var/obj/machinery/body_scanconsole/C = locate(/obj/machinery/body_scanconsole) in range(2,src)
+		if(C)
+			C.connected = src
+		return
```
Comment (PJB3005) [inline code/game/machinery/adv_med.dm]: Again, trailing return.

File: code/game/machinery/adv_med.dm [hunk 4]
```diff
@@ -309,206 +221,366 @@
...
+/obj/machinery/body_scanconsole/proc/findscanner()
+	spawn( 5 )
+		var/obj/machinery/bodyscanner/bodyscannernew = null
+		// Loop through every direction
+		for(dir in list(NORTH,EAST,SOUTH,WEST))
+			// Try to find a scanner in that direction
+			bodyscannernew = locate(/obj/machinery/bodyscanner, get_step(src, dir))
+		src.connected = bodyscannernew
+		return
+	return
```
Comment (PJB3005) [inline code/game/machinery/adv_med.dm]: Trailing return.

**Comment by EmperorJon** [comment]
Can this get a changelog, for convenience? We ought to changelog more things.

**Pull request merged**